src/features/nft/nftModals/AuthorshipClaimModal.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
+import Modal from './modal'
+import Button from 'common/components/Buttons/Button'
+
+export type TAuthorshipClaimModal = {
+  isOpen: boolean
+  handleClose: () => void
+}
+
+const AuthorshipClaimModal: React.FC<TAuthorshipClaimModal> = ({
+  isOpen,
+  handleClose,
+}) => {
+  const data = {
+    nftName: 'i.e banksy168',
+    pastelId: 'i.e banksy168',
+  }
+  const [text, setText] = useState('')
+
+  return (
+    <Modal
```

Comment:
just for future, consider using common/components/AnimatedModal, I think modal looks better with transition

Reply:
Also with current modal the content will always be rendered, even though all modals are closed react will have to spend time to render them